frame/bags-list/Cargo.toml

```
@@ -27,6 +27,8 @@ frame-election-provider-support = { version = "4.0.0-dev", default-features = fa
 
 # third party
 log = { version = "0.4.17", default-features = false }
+docify = "0.2.0"
```

```suggestion
docify = "0.2.1"
```
updated a few days ago to fix oliver's issue